treeherder/webapp/api/runnable_jobs.py

```diff
@@ -54,6 +74,39 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for label, node in tc_graph.iteritems():
+            build_platform = node['task']['extra']['treeherder']['build']['platform']
+            job_type_name = node['task']['metadata']['name']
+
+            # Not all tasks have a group name
+            if 'groupName' in node['task']['extra']['treeherder']:
+                job_group_name = node['task']['extra']['treeherder']['groupName']
+            else:
+                job_group_name = ""
```

Comment:
Another way to write this is:
```python
job_group_name = node['task']['extra']['treeherder'].get(groupName, "")
```
(https://docs.python.org/2/library/stdtypes.html#dict.get)